package.json

```diff
@@ -123,14 +123,15 @@
         "sinon": "^9.2.1",
         "supertest": "^4.0.2",
         "ts-mock-imports": "^1.3.0",
+        "ts-mockito": "^2.6.1",
```

Comment:
what is this for?

Reply:
Had convo offline, but the gist is: 

- Adopt dependency injection (never creating dependencies in our constructors and always getting deps passed in)
- Write unit tests that pass in mocked deps
- Use `ts-mockito` to make making those mocks painless

Reply:
Why not use the already existing TypeMoq?